myia/compile/backends/nnvm.py

```diff
@@ -314,7 +314,7 @@ def convert(self, lst, context):
         target = context.MASK2STR[context.device_type]
         if target == 'cpu':
             nnvm_target = 'llvm'
-        elif target == 'cuda':  # pragma: no cover
+        elif target == 'gpu':
```

Comment:
Why are we using `'gpu'` instead of `'cuda'`?

Reply:
Because that's what tvm uses.